great_expectations/validator/validator.py

```diff
@@ -812,7 +813,7 @@ def resolve_validation_graph(
                     if "metric_calculations" in progress_bars:
                         disable = not progress_bars["metric_calculations"]
 
-            if len(graph.edges) < 3:
```

Comment:
Any reason why we don't set the default to 3? This will change behavior outside of the RBP usecase. I think an argument can be made to keep this value the same.

Reply:
@cdkini I was observing an inconsistency in notebooks, and metrics were not being reported, so one had to wonder if there is a bug.  With reporting them, it makes it easier, and I have not seen any material change in output amount.  Thanks.